Sources/Models and Helpers/ViewModelFetcher.swift

```diff
@@ -16,6 +16,8 @@ class ViewModelFetcher {
         switch widgetV {
```

Comment:
Can we simplify this? Instead of a switch maybe check if responds to viewModel or if conforms to the protocol ?

Reply:
That's the part of the "plumbing code which I despise actually in my design. I have an idea how to make all this better, similar to the idea you had of removing the link between V -> VM. let's discuss this offline